ts-graphql-yoga/README.md

```diff
@@ -0,0 +1,76 @@
+# Graphql Yoga Sample App
+
+## Overview 
+
+The `ts-graphql-yoga` sample application demonstrates how to use the `@klotho::expose` annotation can be used with a graphql-yoga server. The sample application defines a graphql endpoint to run queries and mutations against.
```

Comment:
> how to use the `@klotho::expose` annotation can be used with

Reads weird

Reply:
yeah thats just a bad sentence from doing a lot of readmes, will reword